tools/repl/src/repl.rs

```diff
@@ -229,7 +239,19 @@ impl REPL {
         let mut config = config_file_path
             .as_ref()
             .map(TomlAppServiceConfig::load)
-            .transpose()?
+            .transpose()
+            .map_err(|e| {
+                anyhow!(
+                    "failed to load \"{}\": {}",
+                    config_file_path
+                        .as_ref()
+                        .unwrap_or_else(|| panic!(
+                            "config_file_path is Some because it is used to load file"
```

Comment:
Suggest to remove part about `Some` from error message, it seems not so informative for users.

Reply:
This is an internall error -- a panic. I would write `// unwrap is safe....` instead of this a year ago.  So, the error is not for users, but for devs.